### PR TITLE
DBZ-8044 Add abandoned transaction details to log

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-8044

### CODE REVIEW

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java [hunk 1]
```diff
@@ -1568,9 +1569,10 @@ public void abandonTransactions(Duration retention) throws InterruptedException
                                     }
                                     first = false;
                                 }
-                                LOGGER.warn("Transaction {} (start SCN {}, change time {}, redo thread {}, {} events) is being abandoned.",
+                                LOGGER.warn("Transaction {} (start SCN {}, change time {}, redo thread {}, {} events{}) is being abandoned.",
```
Comment (jpechane) [inline debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java:1572]: There is missing space before `{}` placeholder. Is it intentional?

Comment (Naros) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java:1572]: It is, when the entry has a value it begins with `, ` to be consistent with other listed entries.

**Pull request merged**